compute/messaging/concepts.mdx

```diff
@@ -0,0 +1,188 @@
+
+</Concept>
+
+<Concept>
+
+## Filtering
+
+In a [topic-based](#topic-based) system, where topics handle the logic, filtering is similar to routing. Messages are sent to defined topics, which can be thought of as filters in so far as subscribers can subscribe only to the topics they are interested in. In a [content-based](#content-based) system, filtering is carried out more directly by subscribers, who define filters for messages based on the content/attributes they want to receive.
+
+</Concept>
+
+<Concept>
+
+## Long Polling
+
+Long polling is a technology where the client requests information from the server without expecting an immediate response. In [SQS](#sqs), this enables clients to wait for the system to get messages that are not immediately available.
+
+</Concept>
+
+<Concept>
+
+## Message broker
+
+A message broker is a piece of software that allows applications, systems and services to communicate with each other and send/receive data. It facilitates the exchange of information by receiving messages from a producer, and transmitting them to a consumer. All communication with producers and consumers uses a [protcol](#messaging-protocol). There are two basic models of communication for message brokers: [publish/subscribe](#publishsubscribe) and [queuing](#queuing).
+
+</Concept>
+
+<Concept>
+
+## Messaging
+
+Scaleway's Messaging product is a [message broker](#message-broker) tool that allows you to transfer messages between different microservices and platforms. This enables them to "talk" to each other effectively even if they are not otherwise compatible. Messaging enables and simplifies microservices application development and allows you to build highly scalable, reliable, distributed applications. 
+
+</Concept>
+
+<Concept>
+
+## Messaging protocol
+
+A messaging protocol defines a structured way for users / platforms / services / applications to exchange data and messages, even if normally they do not "speak the same language". Protocols also describe how messages should be processed, prioritized, managed and routed. Messaging protocols currently supported by Scaleway Messaging include [NATS](#nats), [SQS](#sqs) and [SNS](#sns).
+
+</Concept>
+
+<Concept>
+
+## Namespace
+
+A Scaleway Messaging namespace enables systems, applications, and services to communicate with each other and exchange information. A Messaging namespace acts as a message broker, translating messages so that different formal messaging protocols can communicate with one another, even if they are written in different programming languages or implemented on different platforms. Namespaces also allow you to isolate queues and group credentials. Each namespace has a **type**, currently either [NATS](#nats) or [SQS/SNS](#sqssns) which defines the [messaging protocol](#messaging-protocol) to be used.
```

Comment:
```suggestion
A Scaleway Messaging namespace enables systems, applications, and services to communicate with each other and exchange information. A Messaging namespaces allow you to isolate queues and group credentials. Each namespace has a **type**, currently either [NATS](#nats) or [SQS/SNS](#sqssns) which defines the [messaging protocol](#messaging-protocol) to be used.
```

Reply:
+1 for Lucas, the 'translating messages' parts leads to thing different protocols are interoperable, while in fact they're not.